app/objects/secondclass/c_link.py

```diff
@@ -123,6 +123,10 @@ def status(self, value):
             to_status=value
         )
 
+    @classmethod
```

Comment:
This is likely just me missing something, but I'm having trouble thinking of a scenario where a Link would have a global variable associate with it. Abilities definitely, but the links themselves I'm having a bit more trouble with. Is this a 'future proofing' thing?

Reply:
I put this on the `Link` class because it defines a `RESERVED` class-level attribute with `origin_link_id` as a variable and handles the replacement in a command via `apply_id()`.